docs/data/query.md

```diff
@@ -185,6 +185,16 @@ For example, to connect to your Viam organization's MongoDB Atlas instance and q
      [ { numStanding: 215 } ]
      ```
 
+{{< alert title="Tip" color="tip" >}}
+If you use a data field that is named the same as a [reserved SQL keyword](https://en.wikipedia.org/wiki/List_of_SQL_reserved_words), such as `value` or `position`, you must escape that field name in your query using backticks ( <file>\`</file> ).
```

Comment:
> Is it appropriate to link to Wikipedia's standard SQL reserved keywords list, as I have done? (i.e. is it the same list for standard SQL per wiki as for MDB-$SQL?) I was not able to find a MDB resource that talked about reserved or protected keywords. Thanks!

Very good question - so Atlas SQL is based on/mostly follows the SQL-92 standard, per the docs [here](https://www.mongodb.com/docs/atlas/data-federation/query/sql/language-reference/#compatability-and-limitations). We might want to document that somewhere and/or link to the Atlas SQL Dialect Language Reference. It doesn't have a list of reserved keywords in the docs, but [this is the section](https://www.mongodb.com/docs/atlas/data-federation/query/sql/language-reference/#identifiers) of their documentation that I found that talks about reserved keywords. I guess one could also use double quotes, which I hadn't realized and haven't tested.

It would probably be better to link to a list of SQL-92 reserved keywords, but I can't find a good link anywhere. I think linking to the Wikipedia page is okay since as I understand it, SQL:2023 keywords are just a superset of SQL-92, so the list will include any reserved keywords in Atlas SQL (+ some extras). Up to you though, as you are the documentation expert!

Reply:
Will try to balance both options: retaining link to keywords (most helpful resource) with additional link to MDB Atlas data federation docs to further clarify: (SQL-92 + some other possibly-salient limitations). LMK if you agree!

Reply:
I ran into the same issue when I did a quick search - so I agree! Worst case scenario is the customer uses backticks to delimit the identifier when the backticks are not necessary, but the query would still work as expected.